src/components/landing/hero/header/navbar/hero-navbar.view.tsx

```diff
@@ -1,28 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
 
 const HeroNavbarView: React.FC = () => {
+  const [dropMenu, setDropMenu] = useState(false);
+
+  const hideDropMenu = () => {
+    setDropMenu(false);
+  };
+
+  const showDropMenu = () => {
+    setDropMenu(true);
+  };
+
   return (
     <div className="navbar">
-      <div className="navbar__button">
-        <Link to="/" style={{ textDecoration: `none` }}>
-          <p className="navbar__button-text">HOME </p>
-        </Link>
-      </div>
       <div className="navbar__button">
         <Link to="/portal" style={{ textDecoration: `none` }}>
           <p className="navbar__button-text">APPLY</p>
         </Link>
       </div>
-      <div className="navbar__button">
-        <a
-          href="mailto:sponsor@cruzhacks.com"
-          style={{ textDecoration: 'none' }}
+      <div
+        onMouseEnter={showDropMenu}
```

Comment:
nice adaptation here. this will be useful for when we alter the mobile navbar. i'm thinking we should create a re-usable component that you pass a bit of nested json to for what the selections are and some other config.